.gitignore

```diff
@@ -8,3 +8,6 @@ lesson13/__init__.py
 lesson13/json__ponomarov.log
 lesson14/__init__.py
```

Comment:
__init__ files very necessary to be in your PR. It is very important for correct work of your python packages. Please delete this PR and push new one without __init__ files in .ginignore file.

Reply:
Before this, you were asked to remove all unnecessary files including __init __. Try to decide.